### DISCUSSION TITLE
Verify your emails as a sender in Gmail

### DISCUSSION THREAD

**Discussion by rakleed** [community/community#57363] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please verify your emails as a sender in Gmail, so that users can be sure that the letter came from you and not scammers, and your logo is also displayed.

What verification looks like: https://support.google.com/mail/answer/13130196

How to get verified: https://support.google.com/a/answer/10911320

**Comment by ImaniAN** [comment]
Where is the question here ?
